### DISCUSSION TITLE
Input with text-transform uppercase not picking correct value

### DISCUSSION THREAD

**Discussion by sanathko** [cypress-io/cypress#16809] [Component Testing]
I have an html input with style text-transform: uppercase, so whenever a user types a value into it, it turns to uppercase. But when I use cypress to locate this elemen't's value 

using` .should('have.value', FIELD)`, I still get the original value entered in lowercase not the value displayed in the input field, which is in uppercase.

is there a way cypress can handle this kind of a situation?

FYI I am using "cypress": "7.4.0"

**Comment by tit** [comment]
There is currently no way to get the rendered text with JavaScript.
